core/src/test/java/kafka/testkit/KafkaClusterTestKit.java

```diff
@@ -139,14 +139,14 @@ public FaultHandler build(String name, boolean fatal, Runnable action) {
 
     public static class Builder {
         private TestKitNodes nodes;
-        private Map<String, String> configProps = new HashMap<>();
-        private SimpleFaultHandlerFactory faultHandlerFactory = new SimpleFaultHandlerFactory();
+        private final Map<String, Object> configProps = new HashMap<>();
+        private final SimpleFaultHandlerFactory faultHandlerFactory = new SimpleFaultHandlerFactory();
 
         public Builder(TestKitNodes nodes) {
             this.nodes = nodes;
         }
 
-        public Builder setConfigProp(String key, String value) {
+        public Builder setConfigProp(String key, Object value) {
```

Comment:
This change was required to support `Password` type configs used in `MirrorConnectorsIntegrationSSLTest`